appveyor.yml

```diff
@@ -6,12 +6,16 @@ init:
 - ps: (new-object net.webclient).DownloadFile('https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/Latest/dotnet-dev-win-x64.latest.exe', "c:/dotnet-install.exe")
 - ps: Start-Process c:\dotnet-install.exe -ArgumentList "/install","/quiet" -Wait
 - ps: Install-PackageProvider NuGet -Force
-- ps: Install-Module PlatyPS -Force
+- ps: Install-Module platyPS -Force
 install:
 - cmd: git submodule update --init --recursive
 nuget:
   disable_publish_on_pr: true
 build_script:
+- ps: $psversiontable
+- ps: dotnet --version
+- ps: get-packageprovider nuget
+- ps: ipmo platyPS ; get-module -name platyPS
```

Comment:
Yeah... while debugging I got curious about what version all the stuff being used for the environment is (in case that could explain a difference in behavior between local and appveyor), so I added these and thought it would be useful logging info to keep around.